buildmaster/CMS_TTBAR_8TEV_TOT/metadata.yaml

```
@@ -0,0 +1,43 @@
+setname: "CMS_TTBAR_8TEV_TOT"
+
+nnpdf_metadata:
+  nnpdf31_process: "TOP"
+  experiment: "CMS"
+
+arXiv:
+  url: "https://arxiv.org/abs/1603.02303"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1426692"
+hepdata:
+  url: "NA"
+  version: 
+
+version: 1
+version_comment: "NA"
+
+implemented_observables:
+
+  - observable: {description: "total cross section of ttBar", label: '$\sigma_{t\bar{t}}$', units: "$pb$"}
+    observable_name: X-SEC
+    process_type: INC
+    ndata: 1
+    tables: []
+    kinematics:
+      variables:
+        zero: {description: "", label: '$$', units: ""}
+        mt_sqr: {description: "mass of top quark squared", label: '$m_{t}^2$', units: '$GeV^2$'}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics.yaml
+    data_central: data.yaml
+    data_uncertainties:
+      - uncertainties.yaml
+    kinematic_coverage: [zero, mt_sqr, sqrt_s]
+    plotting:
+      dataset_label: '$Total cross section of t\bar{t} at 8 TeV$'
```

```suggestion
      dataset_label: 'Total cross section of $t\bar{t}$ at 8 TeV'
```

Note that if you put text under latex `$` the spaces will be removed.

But (looking at the report) wouldn't it be better to have `r"CMS $\sigma_{\rm{tt}}^{\rm tot}$ 5 TeV"` ? I think it looks better.

---

Yes, I agree with your suggestion, I will change this for all the datasets. Thanks!